api/sql/schema/750$transfer.idle.execute.sql

```diff
@@ -67,7 +67,9 @@ BEGIN TRY
                 FROM
                     [transfer].[sReversal]
                 WHERE
-                    port IN (SELECT value FROM @ports)
+                    port IN (SELECT value FROM @ports) AND
+                    expireTime <= GETDATE() AND
+                    (reverseIssuer = 1 OR (issuerId != ledgerId AND reverseLedger = 1))
```

Comment:
I think `issuerId != ledgerId` should be moved in the view that returns `reverseLedger`